packages/contentful-webhook-handler/src/index.ts

```diff
@@ -90,7 +90,6 @@ const handleWebhook = async (config: LastRevAppConfig, body: any, headers: Recor
             await handlers.contentType(command as ProcessCommand<ContentType>);
             break;
           default:
```

Comment:
I think it's worth keeping at least a log to identify when webhooks are incorrectly set.

Reply:
@maxtechera are you just referring to situations where we should have configured that webhook to ignore that type of event?

Reply:
Yup. That would be it, just want to keep this message somewhere because we currently have something, so instead of removing it just log it.